### ISSUE TITLE
Pauses between tracks when playing ISO

### ISSUE THREAD

**Issue by Jaubr** [sacd-ripper/sacd-ripper#120]
Hello. Thanks for the good program, it makes me happy, but there is one unpleasant problem. I'm creating an ISO image on a Blu-ray player. When listening, at the moment the track ends and a new one begins, a micro pause occurs (for about half a second). This is very annoying on those albums where switching to the next track is filled with music, that is, there is no pause in the recording. Setting pause=0 in sacd_extract.cfg did not help. I've read a lot of forums, but the only thing that is advised is to convert from ISO to DSF. I do not like it. My system supports playback from ISO, besides, when converting to DSF, the original is lost and the sound quality deteriorates slightly. Please tell me, is it possible to make ISO without such pauses? Thank you.

**Comment by Jaubr** [comment]
I forgot to clarify. I use SACDExtractGUI on Windows 10.

**Comment by elvisizer** [comment]
I’ve always been told this is an issue with the playback device/software not being able to do gapless payback from ISO rather than an issue with the rip.

**Comment by Jaubr** [comment]
I have ISO made on Play Station 3 and it play without pause. Why is that? (This is my PS3, I made the image myself from a legally purchased disc.)

**Comment by Jaubr** [comment]
It play without pause on the same player.

**Comment by elvisizer** [comment]
What does iso made on ps3 mean exactly?  Is that an sacd or some other type of iso?  If it was a sacd was it created using sacd extract running on the ps3? Using the ps3 as the transport but connected to sacd extract over the network? What version of sacd extract was used (if that is what was used)?
If you’ve been able to successfully make an sacd iso that plays back gapless in the same playback chain that is now giving you pauses then the problem is the rip…. But all the details about what worked vs what isn’t including settings and software versions will help get us to a solution faster.
And really it is true that you can just avoid all this by making dsf instead of iso!

**Comment by Jaubr** [comment]
Yes, it was exactly SACD ISO. Unfortunately, I can’t give the exact configuration that I set on the PS3 because it was 10 years ago, I don’t remember anymore and I no longer have a PS3. If you have any ideas for what reasons there may be pauses, please write and I will experiment with this (change something in the settings, etc.) so that I know in which direction to move.
By the way, I’ll also try to make ISO through the command line, that is, not through the GUI.
